examples/tutorials/hello-gltf/app.ts

```diff
@@ -0,0 +1,88 @@
+import {AnimationLoopTemplate, AnimationProps, Model} from '@luma.gl/engine/index';
+import {Device, load} from '@loaders.gl/core';
+import {GLTFLoader, postProcessGLTF} from '@loaders.gl/gltf';
+import {createGLTFObjects} from '@luma.gl/gltf/index';
+import {Matrix4} from '@math.gl/core';
+
+const INFO_HTML = `
+Have to start somewhere...
+`;
+
+const lightSources = {
```

Comment:
The shader modules should export typings for their props now. I would recommend importing and using it here.